src/events/common.ts

```diff
@@ -67,7 +67,7 @@ export class AppDiscord {
       })
     })
 
-    schedule.scheduleJob('* * *', async () => {
+    schedule.scheduleJob('*/1 * * *', async () => {
```

Comment:
have u tested it? still looks weird

Reply:
😂 I know, it should work though ... I tested for every minute which is `*/1 * * * *`, per the docs 1hr will be ``*/1 * * *``